apps/test-pololu370-encoder/src/main.cpp

```diff
@@ -0,0 +1,46 @@
+#include "Logger.h"
+#include "Pololu37D.h"
+
+void read_all();
+
+// first pin is yellow wire
+// second pin is white wire
+constexpr Encoder::Pololu37D::Config config = {PA_5, PA_6, NC, 0, GPIO::QEI::Encoding::X2_ENCODING};
+Encoder::Pololu37D enc(config);
+
+Timer theta_timer, theta_dot_timer;
+float theta     = 0;
+float theta_dot = 0;
+
+int main() {
+  while (true) {
+    theta_timer.reset();
+    theta_dot_timer.reset();
+    for (int i = 0; i < 3; i++) {
+      read_all();
+    }
+    enc.reset();
+  }
+}
+
+void read_all() {
+  // angle reading
+  theta_timer.start();
+  if (enc.getAngleDeg(theta)) {
+    theta_timer.stop();
+    Utility::Logger::printf("Angle reading took: %lluus\tAngle: %.3f\r\n", theta_timer.elapsed_time().count(), theta);
+  } else {
+    Utility::Logger::printf("an error occured :(");
```

Comment:
nit: append newlines to the end of print statements

Reply:
added them to the error statements